### PR TITLE
Stuffing early OpenOps draft into README

### PR DESCRIPTION
**Pull request by it33**
Needs tidying up, updated screenshots, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,89 @@
...
+
+- **Phase 4:** Set and implement policies as availability is incrementally rolled out.
+
+## What does OpenOps include? 
+
+Deploying the OpenOps sandbox includes the following componenents: 
+- 🏰 **Mattermost Server** - Open source, self-hosted alternative to Discord and Slack for strict security environments with playbooks/workflow automation, tools integration, real time 1-1 and group messaging, audio calling and screenshare.  
+- 📙 **PostgreSQL** - Database for storing private data from multi-user, chat collaboration discussions and audit history.    
+- 🤖 **Mattermost AI plugin** - Extension of Mattermost platform for AI bot and generative AI integration. 
+- 🦙 **Open Source, Self-Hosted LLM models** - Models for evaluation and use case development via Hugging Face, including GPT4All (runs on a laptop in 4.2 GB) and Falcon LLM (example of leading scaled self-hosted models).
```
Comment (crspeller) [inline README.md:34]: We can point to https://github.com/go-skynet/LocalAI as what we are using for the self host llm models. 

Let's change via Hugging Face to "from Hugging Face and other sources" we don't interact directly with hugging face.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,89 @@
...
+- 🏰 **Mattermost Server** - Open source, self-hosted alternative to Discord and Slack for strict security environments with playbooks/workflow automation, tools integration, real time 1-1 and group messaging, audio calling and screenshare.  
+- 📙 **PostgreSQL** - Database for storing private data from multi-user, chat collaboration discussions and audit history.    
+- 🤖 **Mattermost AI plugin** - Extension of Mattermost platform for AI bot and generative AI integration. 
+- 🦙 **Open Source, Self-Hosted LLM models** - Models for evaluation and use case development via Hugging Face, including GPT4All (runs on a laptop in 4.2 GB) and Falcon LLM (example of leading scaled self-hosted models). 
+- 🧠  **(Optional) Closed Source, Vendor-Hosted AI models** - SaaS-based GenAI models from Azure AI, OpenAI & Anthropic.  
+- 📱 **(Optional) Mattermost Mobile and Desktop Apps** - End user apps for future production deployment. 
+
+## How do I start? 
+
+- 💻 Local Install - [Follow instructions for local installation](https://github.com/mattermost/mattermost-plugin-ai#local).
```
Comment (crspeller) [inline README.md:40]: These should point below.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,89 @@
...
+| [![image](https://github.com/it33/sandbox/assets/177788/6190288e-e624-44f1-aa0f-97d4dc09854c)](https://user-images.githubusercontent.com/3191642/239080952-6fed05e2-ee68-40db-9ee4-870c61ccf5dd.mp4)| **Contextual Interrogation** - Users can ask follow-up questions to discussion summaries generated by AI bots to learn more about the underlying information without reviewing the raw input. | 
+| ![image](https://github.com/it33/sandbox/assets/177788/ddbae9e5-82ef-4ea4-b776-5013f1811047) | **Engage AI Bots like Human Users** - End users can interact with the AI bot in any discussion thread by mentioning AI bot with an `@` prefix, as they would get the attention of a human user. The bot will receive the thread information as context for replying.<br><br> Note: AI bot must be added to the Team before it can be mentioned. |
```
Comment (crspeller) [inline README.md:65]: > AI bot must be added to the Team before it can be mentioned.

There is no adding the bot to the team. This can be removed.

**Review by crspeller — CHANGES_REQUESTED** [review]
Nice start!

**Comment by it33** [comment]
@crspeller I'm mobile today, could I ask your help to just merge the changes you want? 

If there's anything I think differently on I can make another PR

**Review by crspeller — APPROVED** [review]
Applied my fixes.

**Pull request merged**